velox/substrait/tests/VeloxSubstraitRoundTripTest.cpp

```diff
@@ -72,6 +72,38 @@ class VeloxSubstraitRoundTripTest : public OperatorTestBase {
     assertQuery(samePlan, duckDbSql);
   }
 
+  void assertCastPlanConversion(
```

Comment:
Why can't we use assertPlanConversion?

I'd expect that we use assertPlanConversion with 2 plans: one uses regular cast, another uses try_cast.